docs/conf.py

```diff
@@ -14,17 +14,20 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('..'))
+from datetime import date
+
+sys.path.insert(0, os.path.abspath(".."))
```

Comment:
Out of curiosity, why are there so many `'` to `"` changes? Are you using [Black](https://black.readthedocs.io/en/stable/the_black_code_style.html#strings)? Ideally, in a PR, we should aim for minimal noises (i.e. minimize the red-and-green sections).

Reply:
Reverted those changes, I still had auto-format on